### ISSUE TITLE
When using mve wrapper have variables without grad

### ISSUE THREAD

**Issue by IaroslavElistratov** [themis-ai/capsa#124]
> WARNING:tensorflow:Gradients do not exist for variables ['conv2d_37/kernel:0', 'conv2d_37/bias:0'] when minimizing the loss. If you're using `model.compile()`, did you forget to provide a `loss`argument?

**Comment by thearkamitra** [comment]
Hi,
Does this warning change the performance of the model?
